tests/cli-inst-interactive.rs

```diff
@@ -14,6 +14,12 @@ use std::io::Write;
 use std::process::Stdio;
 use std::sync::Mutex;
 
+macro_rules! for_host {
```

Comment:
I don't really feel that a used-once macro that just forms a function once, used in a single test is really pulling its weight - but if you feel it is going to be used lots I'm cool with it being added.

Reply:
I'm intending, once this is merged, to file an e-easy e-mentor help-wanted to move those macros into `mock/clitools.rs`